modular_skyrat/modules/modular_vending/code/clothesmate.dm

```diff
@@ -67,16 +65,18 @@
 		/obj/item/clothing/under/tachawaiian/purple = 5,
 		/obj/item/clothing/under/tachawaiian/green = 5,
 		/obj/item/clothing/under/tachawaiian/blue = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/grey/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/black/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/red/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/tan/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/blue/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/green/skirtybaby = 5,
-		/obj/item/clothing/under/rank/civilian/lawyer/inferno = 3,
-		/obj/item/clothing/under/rank/civilian/lawyer/inferno/skirt = 3,
-		/obj/item/clothing/under/suit/helltaker = 5,
-		/obj/item/clothing/under/suit/helltaker/skirt = 5,
+		/obj/item/clothing/under/suit/skyrat/pencil = 5,
```

Comment:
I can add the pencil-skirts to the loadout once its a bit sorted, I want to have another tab for ONLY formal-suits that are subtypes of under/suit because the 'formal' tab is full of junk right now